### ISSUE TITLE
got bootloop after installing in redmi 5 plus vince w/ miui 10

### ISSUE THREAD

**Issue by nya1998** [rovo89/Xposed#390]
i got a bootloop after try to install xposed in new miui 10

any solution?

**Comment by eebssk1** [comment]
miui need special port version of xposed.just wait.

**Comment by nya1998** [comment]
ok, thanks

**Comment by rastanarcisz** [comment]
yes please help i using redmi 5 plus always bootloop instal xposed

**Comment by nya1998** [comment]
@EBK21 has ben explain that, MIUI need special porting, if you still need
xposed, use custom rom
